README.md

```diff
@@ -139,7 +139,8 @@ This will open up the `xrpackage.js` runtime in your browser and load the given
 See [run.html](https://github.com/webaverse/xrpackage/blob/master/run.html) for the full example.
 
 ```
-import {XRPackageEngine, XRPackage} from 'https://xrpackage.org/xrpackage.js';
+import {XRPackage} from 'https://xrpackage.org/xrpackage.js';
+import {XRPackageEngine} from 'https://xrpackage.org/xrpackage-engine.js';
```

Comment:
from what I've seen, I think `XRPackage` and `XRPackageEngine` are included in a few other places outside of this repo too (e.g https://github.com/search?q=org%3Awebaverse+https%3A%2F%2Fxrpackage.org%2Fxrpackage.js&type=Code).

so I _think_ it might be safer to keep the `XRPackageEngine` export in `xrpackage.js` too, so old code can continue to do stuff like `import { XRPackage, XRPackageEngine } from 'https://xrpackage.org/xrpackage.js'` without having to update all the references.

Reply:
The philosophy here was that the user should only have to import a single file to work with the library.

Maybe that's not great, but that was the reasoning. If we were to go along with that I still think everything should be in its own file, but the top level import (e.g. `xrpackage.js`) could export all of the dependencies, which are in their separate files like `xrpackage-engine.js`.

The other alternative would be to change all of the references to use a multi-file import, but I'm not convinced that is a better user experience.

Reply:
Notably this is part of our official API so it might be a disruptive change.